_includes/head.html

```diff
@@ -23,19 +23,20 @@
         indexUrl: '/js/index.json',   // Url for the .json file containing search index data
         results : '#search-results',  // selector for containing search results element
         entries : '.entries',         // selector for search entries containing element (contained within results above)
+        titleMsg  : '',   // message attached in front of results (can be empty)
         template: '#search-results-template'  // selector for Mustache.js template
       });
     });
   </script>
-  
+
   <script>
     (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
     (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
     m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
     })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
-  
+
     ga('create', 'UA-69605821-1', 'auto');
     ga('send', 'pageview');
-  
```

Comment:
In this case it does not matter, but generally it's preferred not to change `whitespace` characters in commits, unless you're editing something on that line.